config/cloud.cfg.tmpl

```diff
@@ -33,7 +33,7 @@ ssh_pwauth:   0
 # This will cause the set+update hostname module to not operate (if true)
 preserve_hostname: false
 
-{% if variant.endswith("bsd") %}
+{% if not variant.endswith("bsd") %}
```

Comment:
This is incorrect; this stanza is only present to workaround issues seen on BSDs, so it should only be emitted under the condition that was already here.

(If the issue you were fixing was that `RbxCloud` wasn't being found on *BSD, then you'll want to add it to the `datasource_list` that this `if` clause guards.)